Offseason2022/src/main/java/frc/robot/commands/VisionOn.java

```diff
@@ -3,27 +3,35 @@
 
 package frc.robot.commands;
 
+import javax.lang.model.util.ElementScanner6;
+
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.VIConsts;
+import frc.robot.RobotContainer;
 
 /**
  *
  */
-public class ShooterAim extends CommandBase
+public class VisionOn extends CommandBase
 {
   private boolean m_mode;
+  private boolean m_lightOn;
 
-  public ShooterAim(boolean mode)
+  public VisionOn(boolean lightOn)
   {
-    m_mode = mode;
-
-    // m_subsystem = subsystem;
-    // addRequirements(m_subsystem);
+    m_lightOn = lightOn;
+    setName("VisionOn");
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize( )
-  {}
+  {

```

Comment:
This is a robotcontainer reference like this:
RobotContainer.getInstance( ).m_vision.methodInVisionSubsystemToCall();